### ISSUE TITLE
Select Tool, Advanced - Additional CSS Class, Field Focus Loss

### ISSUE THREAD

**Issue by sitedev** [WordPress/gutenberg#21837]
If you're using the new "Select" tool from the tool bar and select a block, then add a class under Advanced > Additional CSS class(es), when you start typing you lose cursor focus after the first letter. If you go back a click the edit tool it works fine.

Editing a settings field doesn't normally require you to go back and change tools, so it's more of a UX issue than an actual bug.

Otherwise I'm delighted with the new, and much needed "Select" tool.

**To reproduce**
1. Using the new "Select" tool.
2. Select any Gutenberg block.
3. Then click "Advanced" in the right hand tool column. 
4. Start typing in the "Additional CSS class(es)" field.
5. After the first character you lose focus on that field.
6. You have to change from the "Select" tool to the "Edit" tool for the field to work correctly.

**Expected behavior**
The "Additional CSS class(es)" field should be editable even if the "Select" tool is currently active. 

**Screenshots**
N/A

**Editor version (please complete the following information):**
- WordPress version: 5.4
- Default Gutenberg Editor
- Gutenberg plugin: no

**Desktop (please complete the following information):**
 - OS: Win7 pro
 - Browser Chrome
 - Version 81.0.4044.113 (Official Build) (64-bit)

**Smartphone (please complete the following information):**
- N/A

**Additional context**
- N/A

**Comment by talldan** [comment]
Thanks for reporting, I can reproduce this.

**Comment by talldan** [comment]
Testing the latest version, this no longer happens, so will close the issue.
